integration-tests/runtime-tests/test/tests/dexTests/constantProductDex/test_configuration.json

```diff
@@ -0,0 +1,5 @@
+{
+  "enabledTests": {
```

Comment:
Quick reminder to add boolean switches for all tests.

Reply:
Thanks for the suggestion, added.